CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master
 
+* Fixed issue where a released reference that was kept in `CLLocationManager` as `unowned` could cause crash. ([#1820](https://github.com/mapbox/mapbox-navigation-ios/pull/1820))
```

Comment:
The dangling reference was in SDK code; mentioning it in the changelog entry makes it sound to the developer like the crash would occur when application code had a dangling reference.

```suggestion
* Fixed a crash during turn-by-turn navigation. ([#1820](https://github.com/mapbox/mapbox-navigation-ios/pull/1820))
```